services/rfq/relayer/service/handlers.go

```
@@ -91,32 +91,43 @@
+		Transaction:   bridgeTx,
+		Status:        reldb.Seen,
+		OriginTxHash:  req.Raw.TxHash,
+	}
+
 	// TODO: you can just pull these out of inventory. If they don't exist mark as invalid.
 	originDecimals, destDecimals, err := r.getDecimalsFromBridgeTx(ctx, bridgeTx)
 	// can't use errors.is here
 	if err != nil && strings.Contains(err.Error(), "no contract code at given address") {
-		logger.Warnf("invalid token, skipping")
+		dbReq.Status = reldb.WillNotProcess
+		dbReq.Reason = fmt.Sprintf("invalid token (no contract code at address)")
```

golangci-lint (1.62.2)

S1039: unnecessary use of fmt.Sprintf

(gosimple)